R/appenders.R

```diff
@@ -426,4 +426,31 @@ appender_async <- function(appender, batch = 1, namespace = 'async_logger',
 
 }
 
+#' Send log messages to a network syslog server
+#' @param identifier program/function identification (string).
+#' @param server machine where syslog daemon runs (string).
+#' @param port port where syslog daemon listens (integer).
+#'
+#' @return A function taking a \code{lines} argument.
+#' @export
+#' @note This functionality depends on the \pkg{syslognet} package.
+#' @examples \dontrun{
+#' if (requireNamespace("syslognet", quietly = TRUE)) {
+#'   log_appender(appender_syslognet("test_app", 'remoteserver'))
+#'   log_info("Test message.")
+#' }
+#' }
+appender_syslognet <- function(identifier, server, port = 601L) {
+  logger::fail_on_missing_package('syslognet')
```

Comment:
Sorry, just saw your PR -- first of all, thanks a lot for your contribution!

Not sure if the double colon is needed here:

```suggestion
  fail_on_missing_package('syslognet')
```

On the other hand, we need to `force` the `identifier`, `server` and `port` params